src/libcrun/linux.c

```diff
@@ -409,6 +409,28 @@ do_mount_setattr (const char *target, int targetfd, uint64_t clear, uint64_t set
   return 0;
 }
 
+static int
+get_bind_mount (const char *src, libcrun_error_t *err)
+{
+  cleanup_close int open_tree_fd = -1;
+  cleanup_close int fd = -1;
```

Comment:
Following variable is unused, causing clang check to fail.

Reply:
fixed